README.md

```diff
@@ -5,3 +5,57 @@ This Module provides functionality for Spinnaker plugins with Go language
 
 ## How To Use This Package
 
+In your Go files, simply use:
+
+```
+import "github.com/armory/plugin-tools/pkg/validation"
+```
+
+### Validations
+
+The only exposed function is:
+
+```
+func ArePluginsCompatible(spinnakerVersion string, plugins []plugin, repos []string) []CompatibilityResult
```

Comment:
I would consider including a versioning "scheme" in the parameter list here, since you want to allow for future versioning schemes and for the Armory versioning system. 

If the list of arguments is getting too long, you could define a struct instead.

Reply:
Also consider changing the name of the function. When I read a function name that starts with "Is" or "Are", I'm expecting a `boolean` result. I might call it `ResolvePluginCompatibility` or something.